clients/src/test/java/org/apache/kafka/clients/admin/KafkaAdminClientTest.java

```diff
@@ -592,6 +597,26 @@ private static ApiVersionsResponse prepareApiVersionsResponseForDescribeFeatures
                 .setErrorCode(error.code()));
     }
 
+    private static QuorumInfo defaultQuorumInfo() {
+        return new QuorumInfo(Topic.METADATA_TOPIC_NAME, 0,
+                singletonList(new QuorumInfo.ReplicaState()),
+                singletonList(new QuorumInfo.ReplicaState()));
+    }
+
+    private static DescribeQuorumResponse prepareDescribeQuorumResponse(Errors error) {
+        if (error == Errors.NONE) {
+            return new DescribeQuorumResponse(DescribeQuorumResponse.singletonResponse(
+                        new TopicPartition(Topic.METADATA_TOPIC_NAME, 0),
+                        0, 0, 0,
+                        singletonList(new DescribeQuorumResponseData.ReplicaState()),
+                        singletonList(new DescribeQuorumResponseData.ReplicaState()))
```

Comment:
It would be better to fully populate the response here. That would have caught the issue that I mentioned earlier.